client/me/security-2fa-code-prompt/index.jsx

```diff
@@ -192,6 +192,7 @@ class Security2faCodePrompt extends React.Component {
 
 	render() {
 		const method = twoStepAuthorization.isTwoStepSMSEnabled() ? 'sms' : 'app';
+		const hasSmsRecoveryNumber = !! twoStepAuthorization.data.two_step_sms_last_four?.length;
```

Comment:
Similar suggestion to [this one](https://github.com/Automattic/wp-calypso/pull/36811/files#r495007558).